src/redux/missions/missionsSlice.jsx

```diff
@@ -15,20 +15,48 @@ export const fetchMissions = createAsyncThunk(
   async (name, thunkApi) => {
     try {
       const res = await axios.get(URL_MISSIONS);
-      console.log(res.data[0]);
       return res.data;
     } catch (err) {
-      return thunkApi.rejectWithValue('all rockets not in the earth');
+      return thunkApi.rejectWithValue('all missions are completed!');
     }
   },
 );
 
+const joinMission = createAsyncThunk('missions/joinMission', async (missionId, API) => {
+  const mId = missionId;
+  return mId;
+});
+
+const leaveMission = createAsyncThunk(
+  'missions/leaveMission',
+  async (missionId, API) => {
+    const mId = missionId;
+    return mId;
+  },
+);
+
 const missionsSlice = createSlice({
   name: 'missions',
   initialState,
   reducers: {},
   extraReducers: (builder) => {
```

Comment:
Hello @Aqurds ,
You need to follow the requirement.
Use the reducer to toggle the property "reserved".

Reply:
Ok. I will. Thanks 👍